### PR TITLE
Table: init filters.

### PR DESCRIPTION
**Pull request by liril-net**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by liril-net** [comment]
#4005

**Comment by Leopoldthecoder** [comment]
测试挂了

**Comment by liril-net** [comment]
修复了 🤗🤗🤗

File: packages/table/src/table.vue [hunk 1]
```diff
@@ -369,6 +369,16 @@
       this.bindEvents();
       this.doLayout();
 
+      // init filters
+      this.store.states.columns.forEach(column => {
+        if (column.filteredValue && column.filteredValue.length) {
+          this.store.commit('filterChange', {
```
Comment (Leopoldthecoder) [inline packages/table/src/table.vue]: 这样会触发一次 `filter-change` 事件吗？

**Pull request merged**